charts/elasticsearch/values.yaml

```diff
@@ -206,6 +206,13 @@ exporter:
   # Restart policy for all containers
   restartPolicy: Always
 
+  podSecurityContext: {}
+  securityContext:
```

Comment:
What is the difference between `podSecurityContext` and `securityContext`?

Reply:
podSecurityContext applies to all pods and containers
containerSecurityContext applies only to containers

Reply:
i need to write up a seperate doc on what policies applies to which service

Reply:
Do we need to update `securityContext` to `containerSecurityContext` in that case?

Reply:
may be on next PR since for this one service if we make change that might create confusion